src/Instance/InstanceDetails/InstanceDetails.js

```diff
@@ -74,109 +78,138 @@ const InstanceDetails = ({
   }, [instance, publicationInfo]);
 
   const accordionState = useMemo(() => getAccordionState(instance, accordions), [instance]);
+  const [helperApp, setHelperApp] = useState();
 
-  return (
-    <Pane
-      {...rest}
-      data-test-instance-details
-      appIcon={<AppIcon app="inventory" iconKey="instance" />}
-      paneTitle={title}
-      paneSub={publicationInfo}
-      dismissible
-      onClose={onClose}
-      actionMenu={actionMenu}
-      defaultWidth="fill"
-    >
-      <TitleManager record={instance.title} />
-
-      <AccordionStatus>
-        <Row end="xs">
-          <Col
-            data-test-expand-all
-            xs
-          >
-            <ExpandAllButton />
-          </Col>
-        </Row>
-
-        <InstanceTitle
-          instance={instance}
-          instanceTypes={referenceData.instanceTypes}
-        />
-
-        <AccordionSet initialStatus={accordionState}>
-          {children}
-
-          <InstanceNewHolding instance={instance} />
-
-          <InstanceAdministrativeView
-            id={accordions.administrative}
-            instance={instance}
-            instanceStatuses={referenceData.instanceStatuses}
-            issuanceModes={referenceData.modesOfIssuance}
-            statisticalCodes={referenceData.statisticalCodes}
-            statisticalCodeTypes={referenceData.statisticalCodeTypes}
-          />
-
-          <InstanceTitleData
-            id={accordions.title}
-            instance={instance}
-            titleTypes={referenceData.alternativeTitleTypes}
-            identifierTypesById={referenceData.identifierTypesById}
-          />
+  const showTagsHelperApp = useCallback(() => {
+    setHelperApp('tags');
+  }, []);
```

Comment:
Isn't `setHelperApp` supposed to be in the dependencies?

Reply:
Also I think we could get rid of `useCallback` here, and call `setHelperApp` inline in `detailsLastMenu` (it's memoized anyway).

Reply:
agree, fixed